css/properties/counter-set.json

```diff
@@ -6,13 +6,13 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/counter-set",
           "support": {
             "chrome": {
-              "version_added": false
+              "version_added": "85"
```

Comment:
Confirmed by testing https://mdn-bcd-collector.appspot.com/tests/css/properties/counter-set on Chrome 84 and 85.